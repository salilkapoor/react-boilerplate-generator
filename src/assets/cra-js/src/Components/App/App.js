import React from 'react';

const App = () => <h1>React Boilerplate App</h1>;

export default App;
