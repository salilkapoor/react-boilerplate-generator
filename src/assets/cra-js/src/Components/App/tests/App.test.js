import React from 'react';
import { shallow } from 'enzyme';

import App from '../App';

const defaultProps = {
  sampleHeading: 'Sample',
};

const getWrapperSetup = (props = {}) => {
  const finalProps = { ...defaultProps, ...props };
  return shallow(<App {...finalProps} />);
};

describe('Scenario 1: Testing the initial sample code', () => {
  let wrapper = '';

  beforeEach(() => {
    wrapper = getWrapperSetup();
  });

  test('Unit Test 1: Should render component correctly', () => {
    wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });

  it('Test one', () => {
    expect(2).toBe(2);
  });
});
