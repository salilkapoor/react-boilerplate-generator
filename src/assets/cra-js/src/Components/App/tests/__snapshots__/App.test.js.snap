// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scenario 1: Testing the initial sample code Unit Test 1: Should render component correctly 1`] = `ShallowWrapper {}`;
